docker/optimism/optimism.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+. "$(realpath $(dirname $0))/../scripts/common.sh"
+
+ROOT="$(get_root_dir)"
+OPTIMISM="${ROOT}/docker/optimism/optimism"
+
+# Deployer's address & private key.
+ADDRESS=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
+PRIVKEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+
+CACHE_DIR=$(obtain_cache_dir $0)
+DEPLOYMENT_DIR="${CACHE_DIR}/deployment"
+DEPLOYMENT_CONFIG_FILE="${CACHE_DIR}/optimism-local.json"
+KEYFILE="${CACHE_DIR}/${ADDRESS}.json"
+
+ensure_keyfile_exists ${PRIVKEY} ${KEYFILE}
+echo Beamer deployer\'s keyfile: ${KEYFILE}
+
+down() {
+    echo "Shutting down the end-to-end environment"
+    make -C ${OPTIMISM}/ops down
+}
+
+up() {
+    echo "Starting the end-to-end environment"
+    docker_compose_file="-f ${OPTIMISM}/ops/docker-compose.yml"
+    docker compose ${docker_compose_file} up --scale relayer=1 -d
```

Comment:
If this doesn't work, why have `--scale relayer=1` at all?

Reply:
Good point, removed.